test/e2e/e2e_test.go

```diff
@@ -361,28 +364,37 @@ func TestLowNodeUtilization(t *testing.T) {
 	}
 	podsBefore := len(podsOnMosttUtilizedNode)
 
-	t.Log("Running LowNodeUtilization strategy")
-	nodeutilization.LowNodeUtilization(
-		ctx,
-		clientSet,
-		deschedulerapi.DeschedulerStrategy{
-			Enabled: true,
-			Params: &deschedulerapi.StrategyParameters{
-				NodeResourceUtilizationThresholds: &deschedulerapi.NodeResourceUtilizationThresholds{
-					Thresholds: deschedulerapi.ResourceThresholds{
-						v1.ResourceCPU: 70,
-					},
-					TargetThresholds: deschedulerapi.ResourceThresholds{
-						v1.ResourceCPU: 80,
-					},
-				},
-			},
+	t.Log("Running LowNodeUtilization plugin")
+
+	handle := &frameworkfake.HandleImpl{
+		ClientsetImpl:                 clientSet,
+		GetPodsAssignedToNodeFuncImpl: getPodsAssignedToNode,
+		PodEvictorImpl:                podEvictor,
+		EvictorFilterImpl: evictions.NewEvictorFilter(
```

Comment:
`s/evictions.NewEvictorFilter(...)/evictorFilter`

No need to create a new evictor filter when there's already one in `evictorFilter`

Reply:
yes, good finding, i will update soon